php/async/Exchange.php

```diff
@@ -363,12 +363,42 @@ public function sleep($milliseconds) {
         });
     }
 
-    public function create_post_only_order($symbol, $type, $side, $amount, $price, $params = array()) {
+    public function create_post_only_order($symbol, $type, $side, $amount, $price, $params=array()) {
         if (!$this->has['createPostOnlyOrder']) {
             throw new NotSupported($this->id . ' create_post_only_order() is not supported yet');
         }
         $array = array('postOnly' => true);
         $query = $this->extend($params, $array);
         return yield $this->create_order($symbol, $type, $side, $amount, $price, $params);
     }
+
+    public function create_stop_order($symbol, $type, $side, $amount, $price = null, $stop_price = null, $params=array()) {
+        if (!$this->has['createStopOrder']) {
+            throw new NotSupported($this->id + 'create_stop_order() ');
```

Comment:
In PHP string concatenations are done with operator `.` (dot), and operator `+` is only used for arithmetic addition. I would also suggest a bit more verbose message:

```JavaScript
throw new NotSupported($this->id + 'create_stop_order() is not supported yet');
```

Reply:
Let's make sure that we fix string concatenations throughout these new methods.

Reply:
You can transpile that js to proper py/php if you place the js code inside any exchange and then `npm run transpile` – you can then copy the py/php implementations from the transpiled exchange files.